libs/remix-url-resolver/tests/test.ts

```diff
@@ -6,32 +6,61 @@ import * as assert from 'assert'
 describe('testRunner', () => {
   describe('# RemixResolve.resolve()', () => {
     describe('* test without AppManager', () => {
-      describe('test example_1 [local imports]', () => {
+      // describe('test example_1 [local imports]', () => {
```

Comment:
If this doesn't apply anymore, that'd be better to explain why in a comment.

Reply:
ok

Reply:
done